js/main.js

```diff
@@ -52,7 +51,7 @@ function getGenreId() {
   xhr.open('GET', 'https://api.jikan.moe/v4/genres/anime?filter=genres');
   xhr.responseType = 'json';
   xhr.addEventListener('load', async function () {
-    function delay(ms) {
+    function delayGetGenreID(ms) {
       return new Promise(resolve => setTimeout(resolve, ms));
     }
     const createUser = id => {
```

Comment:
This still wasn't addressed. This should be renamed to something like `getGenre` rather than `createUser`

Reply:
Oh that function! Renaming the delay function didnt make alot of sense to me lol. Can do!